fpga_interchange/yosys_json.py

```diff
@@ -103,7 +103,31 @@ def make_gnd_cell(cell, module_data, consts):
     return gnd_cell
 
 
-def convert_cell(module_name, module_data, library, libraries, modules,
+def convert_parameters(device, cell, cell_type, property_map):
+    """ Convert cell parameters to match expression type from default. """
+    for name in property_map.keys():
+        definition = device.get_parameter_definition(cell_type, name)
+        if definition is None:
+            # This parameter doesn't have a special definition, don't touch it.
+            continue
+
+        if not definition.is_integer_like():
+            # Non-integer like parameters come from yosys as a string, leave
```

Comment:
You will need to trim a space off the end if one has been added to disambiguate a bitvector (iirc trim a space off the end if it matches `[01xz]* +`)

Reply:
Python already strips:
```
int(' 01101010101010 ', 2)
```

So I think this is okay.

Reply:
But this is for the non-integer-like path - the space(s) are added to disambiguate what is intended to be a string

Reply:
Ok.  Let me update.

Reply:
I think this is fixed, take a look.

Reply:
LGTM